### PR TITLE
22/history

### PR DESCRIPTION
**Pull request by BlackHawk0**
# Description

This PR contains a component that displays a history timeline of the organisation

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
image

image

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 1]
```diff
@@ -19,8 +19,11 @@ import { NextDirective } from './directives/next.directive';
 import { PrevDirective } from './directives/prev.directive';
 
 import { AboutUsRoutingModule } from './about-us.routing';
+import { AboutUsHistoryCarouselComponent } from './components/about-us-history-carousel/about-us-history-carousel.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:22]: fix order of imports

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 2]
```diff
@@ -3,9 +3,11 @@
     <div mainPage class="home-page">
       <elewa-group-elewa-hero></elewa-group-elewa-hero>
       <elewa-group-home-hero-section></elewa-group-home-hero-section>
+      <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
+      <elewa-group-about-us-history-carousel></elewa-group-about-us-history-carousel>
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:7]: This should be in the about us page not the home page

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Make the necessary updates, but good job so far

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 3]
```diff
@@ -33,12 +36,14 @@ import { AboutUsRoutingModule } from './about-us.routing';
...
     AboutUsCultureComponent,
+    AboutUsHistoryCarouselComponent,
     AboutUsPageComponent,
     ElewaAboutUsLocationSectionComponent
   ],
   exports: [
     TeamMembersCarouselComponent,
-    ElewaAboutUsLocationSectionComponent
+    ElewaAboutUsLocationSectionComponent,
+    AboutUsHistoryCarouselComponent
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:46]: No need to export this

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Please implement above feedback

**Comment by BlackHawk0** [comment]
resolved

File: libs/pages/elewa/about-us/src/lib/components/history/history.component.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<p>history works!</p>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/history/history.component.html:1]: Remove template code like this. They are not part of the website

File: libs/pages/elewa/about-us/src/lib/components/about-us-history-carousel/about-us-history-carousel.component.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  <div class="container-1">
+    <h1 class="heading">Our History</h1>
+    <div>
+      <elewa-group-elewa-horizontal-timeline-carousel></elewa-group-elewa-horizontal-timeline-carousel>
+    </div>
+  </div>
+  <div class="container-2">
+    <p>Learn more about </p>
+    <p class="bold">Elewa's social impact.</p>
+  </div>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/about-us-history-carousel/about-us-history-carousel.component.html:11]: You have two components doing the same thing!!

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 6]
```diff
@@ -26,15 +28,17 @@ import { AboutUsRoutingModule } from './about-us.routing';
...
   ],
   declarations: [
     TeamMembersCarouselComponent,
     NextDirective,
     PrevDirective,
     AboutUsCultureComponent,
     AboutUsPageComponent,
-    ElewaAboutUsLocationSectionComponent
+    ElewaAboutUsLocationSectionComponent,
+    AboutUsHistoryCarouselComponent,
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:41]: This component should be added to the about-us-page with it's selector

**Review by IanOdhiambo9 — COMMENTED** [review]
This issue has been completed by pull/240

**Pull request closed without merging**